### PR TITLE
Quad 88 v2

### PR DESCRIPTION
**Pull request by sarvani11**
added additional lines to the navbar I had missed out in previous pull request

### CODE REVIEW

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
should be the same font as the rest of the page:
image

**Comment by jdamerow** [comment]
retest this please

File: Quadriga/src/main/webapp/WEB-INF/views/public/navbar.jsp [hunk 1]
```diff
@@ -3,4 +3,5 @@
 <li ${currentPage == "exploregraph" ? "class=\"active\"" : ""}><a href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/networks">Explore</a></li>
 <li ${currentPage == "statistics" ? "class=\"active\"" : ""}><a href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/statistics">Statistics</a></li>
 <li ${currentPage == "networksearch" ? "class=\"active\"" : ""}><a href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/search">Search</a></li>
- 
\ No newline at end of file
+<li> <a style=" margin:2px 10px; padding:0px 0px 0px 0px; font-size:80%">${project.owner.name} </a>
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/public/navbar.jsp]: This should show in the logged in user, not the owner of the project.

**Comment by jdamerow** [comment]
Why is "logout" bigger than the username?
image

**Comment by jdamerow** [comment]
retest this please

File: Quadriga/src/main/webapp/WEB-INF/views/public/navbar.jsp (hunk 1 above)
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/public/navbar.jsp]: ???

**Comment by sarvani11** [comment]
Do I have to retest it ?

**Comment by rajataggarwal91** [comment]
Reopening the pull request.

**Comment by jdamerow** [comment]
@rajataggarwal91, @sarvani11, this is not done. Please read my comment: "This should show in the logged in user, not the owner of the project."

**Comment by rajataggarwal91** [comment]
Sorry about that. I missed reading out earlier comments. I have fixed the above issue and made sure that it can be merged to develop. Please let us know if there is something else to be done.

**Comment by jdamerow** [comment]
Besides the conflicts. You forgot the little user icon that you had before now (see one of the earlier screenshots):
image
And also, there should be no "logout" link if the user is not logged in. Just don't have any links in that case at all:
image

**Comment by rajataggarwal91** [comment]
All the above issues were fixed. Please review again.

**Comment by jdamerow** [comment]
@sarvani11 , @rajataggarwal91 , now it locks like this:
image

Also, you need to use spring-security for the username. Otherwise as soon as you clikc on "about" or "rpoject blog" or any other menu entry, username and logout button vanish.

**Comment by jdamerow** [comment]
Include:

```
<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
```

 and then you can use

```
<sec:authorize access="isAuthenticated()" >
```

**Comment by jdamerow** [comment]
Currently logged in user:

```
<sec:authentication property="principal.username" />
```

**Comment by rajataggarwal91** [comment]
Made the changes. Still open for comments if you see any bugs.

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
Logging out gives an error message, since you hardcode "quadriga" in the path:
image

**Comment by rajataggarwal91** [comment]
Fixed the above issue. Please review again.

**Pull request merged**